libs/ui/search/src/lib/results-hits-number/results-hits-number.component.ts

```diff
@@ -1,10 +1,13 @@
 import { Component, Input } from '@angular/core'
 
+interface HitsNumberInput {
+  value: number
+}
 @Component({
   selector: 'gn-ui-results-hits-number',
   templateUrl: './results-hits-number.component.html',
 })
 export class ResultsHitsNumberComponent {
-  @Input() hits: number
+  @Input() hits: HitsNumberInput
```

Comment:
why was that required?

Reply:
Because the type was wrong indeed, `hits` is not a number, but an object with a `value` property.
So in the test, giving `component.hits = 10` was wrong I guess.
In the template, it clearly uses `hits.value` so the typing was incorrect somewhere.

Reply:
Weird that this component does not take a number as input. The reasoning behind the wrapping object is unclear. IMO this has to be revisited in the future.

Reply:
We may on some setup have a non exact hit number with stuff like ""there are at least 10000 hits"" See  https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html#track-total-hits

Reply:
Yes but it should be the parent responsibility to handle that, and not the UI component who's responsibility is just to display a number, not a big deal thought for now :)